src/Service/DynamoDb/src/Input/CreateTableInput.php

```diff
@@ -0,0 +1,511 @@
+                    $payload['AttributeDefinitions'][$indices->kf0c51d3]['AttributeType'] = $input->getAttributeType();
+                })($value);
+            }
+        })($this->AttributeDefinitions);
+        $payload['TableName'] = $this->TableName;
+
+        $x = (static function (array $input) use (&$payload, $indices) {
+            $indices->kb46b5e7 = -1;
+            foreach ($input as $value) {
+                ++$indices->kb46b5e7;
+
+                (static function (KeySchemaElement $input) use (&$payload, $indices) {
+                    $payload['KeySchema'][$indices->kb46b5e7]['AttributeName'] = $input->getAttributeName();
+                    $payload['KeySchema'][$indices->kb46b5e7]['KeyType'] = $input->getKeyType();
+                })($value);
+            }
+        })($this->KeySchema);
+
+        $x = (static function (array $input) use (&$payload, $indices) {
+            $indices->kad3ea0f = -1;
+            foreach ($input as $value) {
+                ++$indices->kad3ea0f;
+
+                (static function (LocalSecondaryIndex $input) use (&$payload, $indices) {
+                    $payload['LocalSecondaryIndexes'][$indices->kad3ea0f]['IndexName'] = $input->getIndexName();
+
+                    $x = (static function (array $input) use (&$payload, $indices) {
+                        $indices->k5749469 = -1;
+                        foreach ($input as $value) {
+                            ++$indices->k5749469;
+
+                            (static function (KeySchemaElement $input) use (&$payload, $indices) {
+                                $payload['LocalSecondaryIndexes'][$indices->kad3ea0f]['KeySchema'][$indices->k5749469]['AttributeName'] = $input->getAttributeName();
+                                $payload['LocalSecondaryIndexes'][$indices->kad3ea0f]['KeySchema'][$indices->k5749469]['KeyType'] = $input->getKeyType();
+                            })($value);
+                        }
+                    })($input->getKeySchema());
+
+                    if (null !== $input->getProjection()) {
+                        (static function (Projection $input) use (&$payload, $indices) {
+                            if (null !== $v = $input->getProjectionType()) {
+                                $payload['LocalSecondaryIndexes'][$indices->kad3ea0f]['Projection']['ProjectionType'] = $v;
+                            }
+
+                            $x = (static function (array $input) use (&$payload, $indices) {
+                                $indices->kefae06f = -1;
+                                foreach ($input as $value) {
+                                    ++$indices->kefae06f;
+                                    $payload['LocalSecondaryIndexes'][$indices->kad3ea0f]['Projection']['NonKeyAttributes'][$indices->kefae06f] = $value;
+                                }
+                            })($input->getNonKeyAttributes());
+                        })($input->getProjection());
```

Comment:
I dont understand this: 

> Argument 1 cannot be null, possibly null value provided

We check for null on 399

Reply:
maybe check on null value does not works on methods..

```php
public function getFoo(): ?string
{
  return rand(0,1) === 0 ? null : 'foo';
}
```

and psalm track value on variables only

Reply:
True. 
I can add a variable.